manifests/runtime.yaml

```diff
@@ -1,26 +1,26 @@
 apiVersion: codefresh.io/v1alpha1
 kind: Runtime
 metadata:
-  name: "{{ name }}"
-  namespace: "{{ namespace }}"
+    name: '{{ name }}'
```

Comment:
all the white spaces here were caused by the format-on-save I recently added. I can revert it.

Reply:
no need